docs/index.rst

```diff
@@ -61,7 +61,7 @@ You can capture a message without being bound by an exception:
 Additional Data
 ---------------
 
-You can add additional data to the `Exception.Data <https://msdn.microsoft.com/en-us/library/system.exception.data.aspx>`
+You can add additional data to the `Exception.Data <https://msdn.microsoft.com/en-us/library/system.exception.data.aspx>`_ 
```

Comment:
That is the syntax in Restructured Text to make a link, see https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst